packages/app/components/Queue/QueueListSharedComponents.tsx

```diff
@@ -108,14 +108,9 @@ const DisableQueueButton = styled(QueueInfoColumnButton)`
 `;
 
 const ClearQueueButton = styled(QueueInfoColumnButton)`
-  background-color: #ff8c00;
-  color: white;
-  &:hover,
-  &:focus {
-    color: #ff8c00;
-    background: #fff;
-    border-color: #ff8c00;
-  }
+  color: #da3236;
+  background: #fff;
+  border-color: #da3236;
```

Comment:
i think you could also just do this via the antd button props, instead of manually setting css attributes https://ant.design/components/button/#components-button-demo-danger